### ISSUE TITLE
Add an API to compare colors

### ISSUE THREAD

**Issue by fatcerberus** [spheredev/neosphere#130]
`Color.Red` etc. are implemented as accessors which return a new color object each time.  This allows scripts to manipulate color objects sourced from them without accidentally overwriting the predefined color values.  However, it does has the downside that `Color.Chartreuse === Color.Chartreuse` evaluates to `false`.  That makes it difficult for a game to compare colors, as essentially each color component must be compared separately.

Add an API such as `Color.is()` which would allow colors to be compared more easily:
```js
if (Color.is(c, Color.Chartreuse))
    console.log(`${c.name}: The best color with the best name.`);
else
    console.log(`${c.name} sucks you uncultured swine! *MUNCH*  Oh... I guess you got eaten by the uncultured swine.  Nevermind!`);
```

**Comment by fatcerberus** [comment]
The decision to be made here is whether to make this a static method, e.g. `Color.is(c, Color.Red)` to align with `Object.is()`, or an instance method, e.g. `c.is(Color.Red)`.  The latter is more concise, but the former seems like it might be more idiomatic for JS so I kind of prefer that.

**Comment by fatcerberus** [comment]
Implemented in master, see 1d44a14aa54618741d5f4866b86056015751731a.

I decided on `Color.is()` (a static method of `Color`).  I feel that makes call sites clearer, particularly when the color being compared against is another variable instead of a predefined color.
